### ISSUE TITLE
Cannot find type 'Process' in scope

### ISSUE THREAD

**Issue by mustafa3103** [danger/swift#584]
when we tried to upload your library with SPM, we took this error: "Cannot find type 'Process' in scope". How can we solve that, we tried lots of ways but we could not find anything :)

**Comment by nurirppan** [comment]
same with me, how solve this error. another version too for me

**Comment by bertrandprenat** [comment]
Hello,


I'm getting the same error with Xcode `15.0.1` and Danger `3.18.0`. It looks like the `Process` type couldn't be found in `ShellExecutor.swift`.

Any updates regarding this issue?

Thank you.

**Comment by dessmith** [comment]
Same issue for me today. Is there any updates on this? 
I see there was a release last week so perhaps we're integrating danger incorrectly?

**Comment by SensehacK** [comment]
Same issue for me. `3.18.1`, build using both Rosetta and ARM iOS simulator.

**Comment by orta** [comment]
Yeah, we figured out what this is over on the danger-js repo a bit earlier, https://github.com/danger/danger-js/issues/1448#issuecomment-2111091257 this can be closed when that issue is resolved

**Comment by SensehacK** [comment]
> Yeah, we figured out what this is over on the danger-js repo a bit earlier, [danger/danger-js#1448 (comment)](https://github.com/danger/danger-js/issues/1448#issuecomment-2111091257) this can be closed when that issue is resolved

So should we wait for a new tag for SPM to update or it internally picks up the newer merged commit at `danger-js` 
@orta

**Comment by filipkjamilov** [comment]
Same issue still present, any fix for this ? I am using version `3.18.1`

**Comment by nathanbeaumont** [comment]
I am seeing this Error still, on Xcode 15.1, Swift 5.9 the merge above didn't seem to help alleviate the issue. I am not able to compile just by adding Danger as a dependency.
